hasher-matcher-actioner/requirements-dev.txt

```diff
@@ -1,4 +1,5 @@
 -r requirements.txt
 pytest==6.2.1
 mypy==0.812
-moto
\ No newline at end of file
+black
```

Comment:
What is this change for?

Reply:
If we run `black` formatting on PRs it seems like we should have it as a requirement for developers

Reply:
I thought requirements-dev is only consumed by github actions. 

@schatten is it generally understood that this is the requirements to install if you are a developer?

Reply:
I would have though github actions needs this anyway to run checks?